docs/topics/basic-types.md

```diff
@@ -433,7 +433,7 @@ If a value of character variable is a digit, you can explicitly convert it to an
 fun decimalDigitValue(c: Char): Int {
     if (c !in '0'..'9')
         throw IllegalArgumentException("Out of range")
-    return c.toInt() - '0'.toInt() // Explicit conversions to numbers
+    return c.code - '0'.code // Explicit conversions to numbers
```

Comment:
This is not a conversion anymore, it's rather getting `code` property. 
To convert a digit char to a number it's better to use the dedicated function now: `Char.digitToInt`.

I'd say this example is overall very misleading. It's fine for illustrating `digitToInt`, but not for the `code` property.

Reply:
I've change `code` properties to `digitToInt()`, now it should work fine.

Reply:
No, I mean the entire function `decimalDigitValue` better to be replaced by `Char.digitToInt()`

Reply:
Oh, I see 🤦 
FIxed it.